doc/user/manual_data_access.rst

```diff
@@ -8,20 +8,20 @@ in programmatically building and manipulating inputs and gathering detailed info
 out of ARMI results. Let's now go into a bit more detail for the power user.
 
 Settings and State Variables
-----------------------------
+============================
 The following links contain large tables describing the various global settings
 and state parameters in use across ARMI.
 
-* :doc:`Table of all global settings </user/inputs/settings_report>`
-* :doc:`Reactor Parameters </user/reactor_parameters_report>`
-* :doc:`Core Parameters </user/core_parameters_report>`
-* :doc:`Component Parameters </user/component_parameters_report>`
-* :doc:`Assembly Parameters </user/assembly_parameters_report>`
-* :doc:`Block Parameters </user/block_parameters_report>`
+* :doc:`Table of all global settings <../user/inputs>`
+* :doc:`Reactor Parameters <../user/reactor_parameters_report>`
+* :doc:`Core Parameters <../user/core_parameters_report>`
+* :doc:`Component Parameters <../user/component_parameters_report>`
+* :doc:`Assembly Parameters <../user/assembly_parameters_report>`
+* :doc:`Block Parameters <../user/block_parameters_report>`
```

Comment:
These links don't appear to be working when building the internal documentation.

Reply:
Okay, I fixed that using `:ref:` commands.